src/dso_api/dynamic_api/filterset.py

```diff
@@ -32,11 +50,11 @@ def filterset_factory(model: Type[DynamicModel]) -> Type[DynamicFilterSet]:
     # Determine which fields are included:
```

Comment:
In  VSD https://github.com/Amsterdam/various_small_datasets/blob/master/src/various_small_datasets/gen_api/views.py staat een voorbeeld voor generiek geofilter

Reply:
Ohh nice! Voeg ik toe!